commands/audit/jas/applicability/applicabilitymanager.go

```diff
@@ -40,22 +41,29 @@ type ApplicabilityScanManager struct {
 // map[string]string: A map containing the applicability result of each XRAY CVE.
 // bool: true if the user is entitled to the applicability scan, false otherwise.
 // error: An error object (if any).
-func RunApplicabilityScan(xrayResults []services.ScanResponse, directDependencies []string,
-	scannedTechnologies []coreutils.Technology, scanner *jas.JasScanner, thirdPartyContextualAnalysis bool) (results []*sarif.Run, err error) {
-	applicabilityScanManager := newApplicabilityScanManager(xrayResults, directDependencies, scanner, thirdPartyContextualAnalysis)
+func RunApplicabilityScan(auditParallelRunner *utils.AuditParallelRunner, xrayResults []services.ScanResponse, directDependencies []string,
+	scanner *jas.JasScanner, thirdPartyContextualAnalysis bool, extendedScanResults *utils.ExtendedScanResults, module jfrogappsconfig.Module, threadId int) (err error) {
+	var scannerTempDir string
+	if scannerTempDir, err = jas.CreateScannerTempDirectory(scanner, string(utils.Applicability)); err != nil {
+		return
+	}
+	applicabilityScanManager := newApplicabilityScanManager(xrayResults, directDependencies, scanner, thirdPartyContextualAnalysis, scannerTempDir)
 	if !applicabilityScanManager.cvesExists() {
-		log.Debug("We couldn't find any vulnerable dependencies. Skipping....")
+		log.Debug("[thread_id: " + strconv.Itoa(threadId) + "] We couldn't find any vulnerable dependencies. Skipping....")
 		return
 	}
-	if err = applicabilityScanManager.scanner.Run(applicabilityScanManager); err != nil {
+	log.Info("[thread_id: " + strconv.Itoa(threadId) + "] Running applicability scanning...")
+	if err = applicabilityScanManager.scanner.Run(applicabilityScanManager, module); err != nil {
```

Comment:
maybe after running you can add a log for how many applicable & not applicable discovered, just like the other scans